Scroller/EditableWrapper.js

```diff
@@ -687,20 +687,43 @@ const EditableWrapper = (props) => {
 	}, [completeEditingByKeyDown]);
 
 	const handleGlobalKeyDownCapture = useCallback((ev) => {
-		if (getPointerMode() && !scrollContainerRef.current.contains(Spotlight.getCurrent()) && (mutableRef.current.selectedItem || mutableRef.current.focusedItem)) {
+		const {focusedItem, selectedItem} = mutableRef.current;
+
+		// If the pointer mode is `true` and the focused component is not contained in scrollContainerRef,
+		// only `handleGlobalKeyDownCapture` is called instead of `handleKeyDownCapture`
+		if (getPointerMode() && !scrollContainerRef.current.contains(Spotlight.getCurrent()) && (selectedItem || focusedItem)) {
 			const {keyCode} = ev;
 			const position = getLastPointerPosition();
 			const direction = getDirection(keyCode);
 			if (direction) {
 				const nextTarget = getTargetByDirectionFromPosition(direction, position, mutableRef.current.spotlightId);
 
 				if (!scrollContainerRef.current.contains(nextTarget)) {
+					// If the nextTarget is not contained in scrollContainerRef, complete editing
 					const orders = finalizeOrders();
 					finalizeEditing(orders);
+				} else if ((is('left', keyCode) || is('right', keyCode)) && selectedItem) {
+					// When an item is selected and press the `left` or `right` key, move the selectedItem in that direction
+					moveItemsByKeyDown(ev);
```

Comment:
As we discussed, the focus seems to move after this. Let's prevent the moving focus.

Reply:
It's fixed.